sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -397,6 +397,9 @@ def test_register_finalizations(self):
   def test_custom_merging_window(self):
     raise unittest.SkipTest("BEAM-11004")
 
+  def test_pack_combiners(self):
+    raise unittest.SkipTest("BEAM-12305")
```

Comment:
Could you fill in the details on this bug exactly what the failures are?

Reply:
Updated with bug description.